.github/feature_request.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FR]"
+labels: ["🚦 status: in discussion", "🏗️ type: new feature"]
+assignees: ''
+
+---
+
+**Please give information what category your feature request is about**
+
+Examples:
+- An extension
+- The build or infrastructure (IDE, Github, Gradle)
+- The documentation
+- Something different
+
+**Please describe the cause of your feature request? What do you try to do and realize that a tool / functionality is missing?**
+
+For example:
+- I tried to use extension Y to do X, but with the current implementation I can only do Z
+- I read about this fancy thing, Gradle can do now!
+
+**Please describe, what you would like to be changed/ added / improved. If you want to add code or pseudo-code snippets, feel free to do so. If you have knowledge about possible side effects or problems or have considered any alternative solutions, please add information about them, too**
+
+Example:
+- I would like to see the possibility to use a new annotation, maybe `@fancy("value")` to do X
+- I suggest to update to Java 2042 to use the new feature Z, but this also means, that the Gradle version has to be updated, and it's only a RC so far.
```

Comment:
You didn't use a period for the other unordered list elements. Also, "RC" might not be obvious to everybody.

```suggestion
- I suggest to update to Java 2042 to use the new feature Z, but this also means, that the Gradle version has to be updated, and it's only a release candidate (RC) so far
```